src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -1830,6 +1839,7 @@ public void onAction(Action action, ReactMouseEvent event) {
         onSubmit();
         break;
       case UPLOAD_NEW_FILE:
+      case UPLOAD_FILE:
         onUploadFile();
```

Comment:
These were different at one point, but today they both open the same modal with the same props